src/test/java/lotto/domain/WinningLottoTest.java

```diff
@@ -0,0 +1,17 @@
+package lotto.domain;
+
+import lotto.exceptions.DuplicateLottoNumberException;
+import lotto.pattern.TestNumberGenerator;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class WinningLottoTest {
+    @Test
+    @DisplayName("이번주 우승 로또 번호와, 보너스볼의 번호는 중복이 될 수 없다.")
+    void winningLotto() {
+        assertThatThrownBy(() -> new WinningLotto(new LottoTicket(new TestNumberGenerator().generateLottoNumbers()), new LottoNumber(6)))
+                .isInstanceOf(DuplicateLottoNumberException.class);
```

Comment:
```suggestion
        List<LottoNumber> lottoNumbers = Arrays.asList(
                new LottoNumber(1),
                new LottoNumber(2),
                new LottoNumber(3),
                new LottoNumber(4),
                new LottoNumber(5),
                new LottoNumber(6)
        );
        assertThatThrownBy(() -> new WinningLotto(new LottoTicket(lottoNumbers), new LottoNumber(6)))
                .isInstanceOf(DuplicateLottoNumberException.class);
```


위와 같은 경우에는 TestNumberGenerator 를 사용하지 말고,
바로 하드 코딩된 generateLottoNumbers 를 직접 명시하는게 가독성이 좋지 않을까 제안해봅니다.
필요한게 "로또 번호" 인지와, "NumberGenerator" 인지는 엄연 다릅니다.
그리고 의존성을 줄일 수 있다면 테스트에서도 의존성을 줄이는 것을 저는 좋아하는 편입니다.